bears/general/LineContinuationBear.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.bearlib.languages.Language import Language
+
+
+class LineContinuationBear(LocalBear):
+    def run(self,
+            filename,
+            file,
+            language,
+            ):
+        """
+        It is a generic bear which will look for any lines which
+        end with the explicit use of line continuation operators,
+        which is considered as a bad practice. The solution
+        would be to use implicit type of line continuation methods.
+
+        Note that this requires the coalang to know the tokens
+        used for line continuation and hence any language not
+        having the line continuation attribute will result in an error.
+
+        :param filename:
+            Name of the file that needs to be checked.
+        :param file:
+            File that needs to be checked in the form of a list of strings.
+        :param language:
+            Language to be used for finding the line continuation tokens.
+        """
+        try:
+            lang = Language[language]
+            line_continuation = lang.get_default_version().line_continuation
+        except Exception:
```

Comment:
The exact exception like the `KeyError` or whatever pops up would be nice over here

Reply:
Hmm, true. Will find which type of exceptions I will get.

Reply:
what if the language is supported but the token for line continuation isn't there? A different error message or the same one for the both? Gonna keep this open for other reviewers ;)

Reply:
Yeah, I had implemented two different error messages. But I felt it was redundant, so just kept one for both since only one language supports line_continuation, Python.

Reply:
It still would be better to have both kinds of exceptions listed here. Everything else looks great!

Reply:
Thank you for the help. I will definitely improve this :smile: